get_ci_state.sh

```diff
@@ -36,6 +36,7 @@ do
   pipeline_id_attempt=$((pipeline_id_attempt + 1))
   if [ "$pipeline_id_attempt" -gt "5" ]
   then
+    echo "Failed to fetch pipeline id"
```

Comment:
I would suggest to also include the used SHA in the output. This allows to double-check if the issue arises from a wrong/strangely chosen SHA. It furthermore allows for easier checking in the remote repo, as the commit-SHA in question does not need to be searched for.

```suggestion
    echo "Failed to fetch pipeline id for ${used_sha}"
```